src/Core/Extensible.php

```diff
@@ -18,7 +17,18 @@
  */
 trait Extensible
 {
-    use CustomMethods;
+    use CustomMethods {
```

Comment:
The other thing that might be worth doing (not sure of perf impact) would be replacing the `empty` calls with a basic if, and ensure that these values are set to true/false in the config rather than omitted.